server.c

```diff
@@ -123,8 +123,37 @@ int main(void)
             if (status == 0) printf("Other connection closed");
             if (status == -1) perror("recv");
             printf("%s", str);
-            if (send(new_fd, "Hello, world!", 13, 0) == -1)
-                perror("send");
+            
+	    FILE* home_pg = fopen("index.html", "r");
+	    if(fseek(home_pg, 0L, SEEK_END) == -1) {
+	    	perror("file seek failed");
+		return 1;
```

Comment:
For each of these that you added, we should use exit(1) instead of return 1 since this will be in the child process so we will always want to exit the process. Right now this works because we are in the main function... but if we move these out into their own functions (which seems likely), then using return instead of exit would be a bug. Let's get it out of the way now so we don't accidentally create a bug later.